### PR TITLE
Update rubocop & rubocop-rspec gems

### PR DESCRIPTION
**Pull request by Watson1978**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -21,9 +21,9 @@ jobs:
       uses: actions/cache@v1
       with:
         path: ./vendor/bundle
-        key: v1-linux-2.3
+        key: v1-linux-2.3-{{ checksum rmagick.gemspec }}
```
Comment (mockdeep) [inline .github/workflows/ci.yml]: @Watson1978 why did you need to add `checksum` here?

Comment (Watson1978) [reply .github/workflows/ci.yml]: To drop cache and install new version, it is useful to check changing its file.

Comment (mockdeep) [reply .github/workflows/ci.yml]: Doesn't `bundle` install any missing dependencies either way? I think this might slow down the build any time `gemspec` changes.
